### ISSUE TITLE
[FR] Kanban Board Line Numbers

### ISSUE THREAD

**Issue by shuen2023** [AppFlowy-IO/AppFlowy#6961]
### Description

Request to add row numbers to Kanban Board column type. Automatically generate corresponding row numbers for each added row.

### Impact

Users who like to use Kanban Board

**Comment by annieappflowy** [comment]
Did you mean a count next to each group name?
<img width="845" alt="image" src="https://github.com/user-attachments/assets/8b51c072-d6d2-42a5-ad32-315852ef52fd">

**Comment by shuen2023** [comment]
> 您是指每个组名旁边的计数吗？ <img alt="图像" width="845" src="https://private-user-images.githubusercontent.com/12026239/394634003-8b51c072-d6d2-42a5-ad32-315852ef52fd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM5MDY3NTYsIm5iZiI6MTczMzkwNjQ1NiwicGF0aCI6Ii8xMjAyNjIzOS8zOTQ2MzQwMDMtOGI1MWMwNzItZDZkMi00MmE1LWFkMzItMzE1ODUyZWY1MmZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEyMTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMjExVDA4NDA1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUxMTk0OWU3MjU5MzIyNTQ4ZGZhYTA3MTJjYzQyZWE0NTcwMzdiYzJmYjYxOWIyOGViM2UzYjgxOWU3Zjk0NDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.uhpgR0E5_9lR1WP_iDqzKTF9NkhhuZuFzkCh8vlTs-E">

No. It is similar to the row numbers in Excel. When I use kanban, row numbers are automatically generated for each additional row.   

![IMG_20241211_164551](https://github.com/user-attachments/assets/bc03f0ef-013c-473f-9aff-98fd16df07d9)
![IMG_20241211_164531](https://github.com/user-attachments/assets/273cb13b-3342-4032-b640-24551a623946)

**Comment by annieappflowy** [comment]
#6652 
duplicate
